bin/stress-test/src/main.rs

```diff
@@ -0,0 +1,471 @@
+    "account_storage_slot_updates",
+    "transactions",
+    "accounts",
+];
+
+#[derive(Parser)]
+#[command(version)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Subcommand)]
+pub enum Command {
+    /// Create and store blocks into the store. Create a given number of accounts, where each
+    /// account consumes a note created from a faucet.
+    SeedStore {
+        /// Path to the store database file.
+        #[arg(short, long, value_name = "DUMP_FILE", default_value = "./miden-store.sqlite3")]
+        dump_file: PathBuf,
+
+        /// Number of accounts to create.
+        #[arg(short, long, value_name = "NUM_ACCOUNTS")]
+        num_accounts: usize,
+
+        /// Path to the genesis file of the store.
+        #[arg(short, long, value_name = "GENESIS_FILE")]
+        genesis_file: PathBuf,
+    },
+}
+
+const BATCHES_PER_BLOCK: usize = 16;
+const TRANSACTIONS_PER_BATCH: usize = 16;
+
+/// Create and store blocks into the store. Create a given number of accounts, where each account
+/// consumes a note created from a faucet. The cli accepts the following parameters:
+/// - `dump_file`: Path to the store database file.
+/// - `num_accounts`: Number of accounts to create.
+/// - `genesis_file`: Path to the genesis file of the store.
+#[tokio::main]
+async fn main() {
+    let cli = Cli::parse();
+
+    match &cli.command {
+        Command::SeedStore { dump_file, num_accounts, genesis_file } => {
+            seed_store(dump_file, *num_accounts, genesis_file).await;
+        },
+    }
+}
+
+/// Seed the store with a given number of accounts.
+async fn seed_store(dump_file: &Path, num_accounts: usize, genesis_file: &Path) {
+    let store_config = StoreConfig {
+        database_filepath: dump_file.to_path_buf(),
+        genesis_filepath: genesis_file.to_path_buf(),
+        ..Default::default()
+    };
+
+    // Start store
+    let store = Store::init(store_config.clone()).await.context("Loading store").unwrap();
+    task::spawn(async move { store.serve().await.context("Serving store") });
+    let channel = tonic::transport::Endpoint::try_from(store_config.endpoint.to_string())
+        .unwrap()
+        .connect()
+        .await
+        .unwrap();
+    let store_api_client = ApiClient::with_interceptor(channel, OtelInterceptor);
+    let store_client = StoreClient::new(store_api_client);
+    let start = Instant::now();
+
+    // Create faucet
+    println!("Creating new faucet account...");
+    let genesis_header = store_client.latest_header().await.unwrap();
+    let faucet_id = create_faucet(&genesis_header);
+
+    // Create first block with the faucet
+    let txs =
+        vec![
+            MockProvenTxBuilder::with_account(faucet_id, Digest::default(), Digest::default())
+                .output_notes(vec![])
+                .build(),
+        ];
+    let batch = ProvenBatch::mocked_from_transactions(txs.iter());
+
+    apply_block(vec![batch], &store_client).await;
```

Comment:
Does the "first block" mean the genesis block here? If so, isn't the genesis block created on startup (i.e., we don't need to insert it into the store explicitly)?

If this is not a genesis block, then why do we need to insert the faucet into it (the faucet should be present in the genesis block)?